src/scss/components/email/_index.scss

```diff
@@ -0,0 +1,184 @@
+@import "components/form/config";
+
+hx-email-control {
+  $ctrl-columns: auto 1fr auto;
+  $ctrl-rows: auto auto;
+  $ctrl-row-align: center;
+
+  -ms-grid-columns: $ctrl-columns;
+  -ms-grid-rows: $ctrl-rows;
+  align-items: $ctrl-row-align;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "label  label   label"
+    "prefix control suffix"
+    "help   help    help"
+    "error  error   error";
+  grid-template-columns: $ctrl-columns;
+  grid-template-rows: $ctrl-rows;
+
+  // blacklist
+  > * {
+    display: none;
+  }
+
+  // whitelist
+  > label,
+  > input[type="email"],
+  > .hxPrefix,
+  > .hxSuffix {
+    display: block;
+  }
+
+  > label {
+    $justify: start;
+
+    -ms-grid-column-align: $justify;
+    -ms-grid-column-span: 3;
+    -ms-grid-column: 1;
+    -ms-grid-row: 1;
+    grid-area: label;
+    justify-self: $justify;
+  }
+
+  > input[type="email"] {
+    -ms-grid-column: 2;
+    -ms-grid-row: 2;
+    grid-area: control;
+    height: 2rem;
+    min-width: 8rem; // ~16 chars
+    padding: 0 0.75rem;
+  }
+
+  > .hxPrefix {
+    $justify: end;
+
+    -ms-grid-column-align: $justify;
+    -ms-grid-column: 1;
+    -ms-grid-row-align: $ctrl-row-align; // because IE
+    -ms-grid-row: 2;
+    grid-area: prefix;
+    justify-self: $justify;
+    margin-right: 0.25rem;
+  }
+
+  > .hxSuffix {
+    $justify: start;
+
+    -ms-grid-column-align: $justify;
+    -ms-grid-column: 3;
+    -ms-grid-row-align: $ctrl-row-align; // because IE
+    -ms-grid-row: 2;
+    grid-area: suffix;
+    justify-self: $justify;
+    margin-left: 0.25rem;
+  }
+
+  > p,
+  p.hxHelpText {
+    $justify: start;
+
+    -ms-grid-column-align: $justify;
+    -ms-grid-column-span: 3;
+    -ms-grid-column: 1;
+    -ms-grid-row: 3;
+    grid-area: help;
+    justify-self: $justify;
+  }
+
+  > p.hxErrorText {
+    $justify: start;
+
+    -ms-grid-column-align: $justify;
+    -ms-grid-column-span: 3;
+    -ms-grid-column: 1;
+    -ms-grid-row: 4;
+    grid-area: error;
+    justify-self: $justify;
+  }
+}
+
+// ----- PRISTINE ----------
+hx-email-control {
+  > label {
+    @include hxFieldName();
+  }
+
+  > input[type="email"] {
+    @include hxTextControl;
+
+    &:focus {
+      @include hxTextControl($focused: true);
+    }
+  }
+}
+
+> p {
+  margin-top: inherit;
+  @include hxHelpText();  // default styles
+
+  .hxHelpText {
+    @include hxHelpText();
+  }
+
+  .hxErrorText {
+    color: $red-900;
+    font-size: 0.75rem;
+  }
+}
+
+.hxOptional {
+  &::after {
+    color: $gray-700;
+    content: " (optional)";
+  }
+}
+
+.hxErrorText {
+  color: $red-900;
+  font-size: 12px;
+  margin-left: inherit;
+}
+
+hx-checkbox-control {
+  > input[type="text"]:focus {
+    border: 1px solid $cyan-700;
+    box-shadow: $focus-glow;
+    outline: 0;
+  }
+}
+
+.align {
+  display: inline;
+}
+
+.hxHelpText {
+  margin-left: inherit;
+}
```

Comment:
We should not need these styles.  We can discuss usage if needed.

Please remove.